packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -438,23 +448,16 @@ void WebView::InitWebView() {
   }
   float scaleFactor = 1;
```

Comment:
There is some routine to calculate the scale factor on the engine side, Should it be shared by the engine? 
This is just my curiosity.

Reply:
I guess.. we should use engine's value. we need to modify platform view's interface for that.